pycaret/internal/pycaret_experiment/time_series_experiment.py

```diff
@@ -2024,9 +2024,30 @@ def predict_model(
 
         data = None  # TODO: Add back when we have support for multivariate TS
 
+        X_test_ = self.X_test.copy()
+        # Some predict methods in sktime expect None (not an empty dataframe as
+        # returned by pycaret). Hence converting to None.
+        if X_test_.shape[0] == 0 or X_test_.shape[1] == 0:
+            X_test_ = None
+        y_test_ = self.y_test.copy()
+
         if fh is None:
             fh = self.fh
 
+        display = None
+        try:
+            np.random.seed(self.seed)
+            if not display:
+                display = Display(
+                    verbose=verbose,
+                    html_param=self.html_param,
+                )
+        except:
```

Comment:
can we catch here the expected error instead of catching all kind of errors ?

Reply:
I have not delved into this part. This is copied over from the regression module. @Yard1 any thoughts on this?

Reply:
It would be a good idea, but I can't remember what sort of an error can pop up there